bayesian/api/api_v2.py

```diff
@@ -303,6 +305,24 @@ def stack_analyses():
         raise HTTPError(500, e.args[0])
 
 
+@api_v2.route('/stack-report/<crda_key>/<stack_id>', methods=['GET'])
+def stack_report_url(crda_key, stack_id):
+    """URL redirect for Stack Report UI."""
+    path = f"{STACK_REPORT_UI_HOSTNAME}/#/analyze/{stack_id}"
+    query_params = "?interframe=true&api_data=" + json.dumps({
+        "access_token": "undefined",
+        "route_config": {
+            "api_url": BAYESIAN_API_HOSTNAME,
+            "ver": "v3",
+            "uuid": crda_key
+        },
+        "user_key": THREESCALE_ACCOUNT_SECRET
+    })
+    final_path = path + query_params
+    logger.info("Redirected to URL: %s ", final_path)
```

Comment:
Can this be a debug print? we already have access logs enabled from gunicorn

Reply:
No, I don't think so, This logs redirect URL, which I feel is not logged by Gunicorn

Reply:
I understand that, what I meant here is that we have minimal info print already in the form access logs for request.

Reply:
@arajkumar This is a really helpful log, I don't think we should skip it.. It logs which URL is built by combining a number of variables.